### DISCUSSION TITLE
Symlinks not being recognized

### DISCUSSION THREAD

**Discussion by iamthetot** [felddy/foundryvtt-docker#1049] [Q&A] [answered]
### Bug description

Within /Data/ I have made a symlink to a /shared/ folder that is located elsewhere. The symlink works in all other file browsers, but when I spin up the foundry server, /shared/ does not show up inside User Data in the file browser.

### Steps to reproduce

1. Create a folder somewhere outside of /Data/ and place some assets into it. Let's call it "shared"
2. Create a symlink with ln -s inside of /Data/ pointing to that "shared" folder.
3. Test to make sure the symlink is working in general.
4. Start the docker container.
5. Inside of Foundry, open the file browser (for example choosing a scene image, or token image, or etc) and navigate to User Data folder. "Shared" folder will not be shown there.

### Expected behavior

I use this exact same setup with two Foundry servers hosted on a Raspberry Pi right now, so I would expect the symlink to work within Foundry. I am left to assume that there's something going on with this container that's preventing the symlink from working inside Foundry.

### Container metadata

```shell
com.foundryvtt.version = "12.331"
org.opencontainers.image.authors = "markf+github@geekpad.com"
org.opencontainers.image.created = "2024-08-16T01:48:32.610Z"
org.opencontainers.image.description = "An easy-to-deploy Dockerized Foundry Virtual Tabletop server."
org.opencontainers.image.licenses = "MIT"
org.opencontainers.image.revision = "61a1ac11d7bd298b93fa6cebb102158ddb1d020e"
org.opencontainers.image.source = "https://github.com/felddy/foundryvtt-docker"
org.opencontainers.image.title = "foundryvtt-docker"
org.opencontainers.image.url = "https://github.com/felddy/foundryvtt-docker"
org.opencontainers.image.vendor = "Geekpad"
org.opencontainers.image.version = "12.331.0"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felddy** [comment, marked as answer]
Docker cannot access files that are not mounted into the container.  If your symbolic link points to a nonexistent location (from the container's point of view) you will get the behavior you are experiencing.  You can however you can mount your shared assets into both containers using additional volume mounts.  

See:
- https://github.com/felddy/foundryvtt-docker/discussions/166
- https://github.com/felddy/foundryvtt-docker/discussions/216

**Reply by iamthetot** [reply]
Hey, thanks for the quick response. Those links look useful. I'm new to docker containers so thanks for the info. I'm working on migrating my existing solution (two servers hosted on a Pi) to an unraid server. I'm not sure how I can add additional volume mounts to this deployment but I will definitely be investigating.
